grammars/css.cson

```diff
@@ -561,6 +561,44 @@
         'captures':
           '1':
             'name': 'punctuation.definition.entity.css'
+          '2':
+            'name': 'entity.other.namespace-prefix.css'
+          '3':
+            'name': 'punctuation.separator.css'
+          '4':
+            'name': 'entity.other.attribute-name.css'
```

Comment:
I noticed you dropped the extra `attribute` from the end of this.  Was that for consistency?